staging/src/k8s.io/csi-api/pkg/apis/csi/v1alpha1/README.md

```diff
@@ -0,0 +1,32 @@
+# CSINodeInfo and CSIDriverInfo Usage and Lifecycle
+CSINodeInfo is an API object representing CSI Information at the Node level.
+CSIDriverInfo is an API object that represents information about a driver at the
```

Comment:
`CSIDriverInfo` isn't exactly a top-level API object, but not sure what's the appropriate name here... sub type?